packages/workspace/src/browser/workspace-service.ts

```diff
@@ -368,7 +368,7 @@ export class WorkspaceService implements FrontendApplicationContribution {
             } catch (error) {
                 // Fall back to reloading the current window in case the browser has blocked the new window
                 this._workspace = uri;
-                this.logger.error(error.toString()).then(async () => await this.reloadWindow());
```

Comment:
no need to have async

Reply:
let's call this "good measure", it is actually better like `async () => this.return_a_promise()` instead of `() => this.return_a_promise()` in case where `this.return_a_promise()` fails before returning a promise (case when it is a regular function (non-async) and it throws before returning said promise).

Reply:
would you like me to fix in this PR before merging?

Reply:
please

Reply:
no problem! done! :)